Broadcom/symantec-endpoint-protection/ingest/parser.yml

```
@@ -47,13 +65,28 @@ stages:
           observer.product: "Symantec Endpoint Protection"
           event.kind: "event"
           event.category: ["malware"]
-          event.type: ["info"]
           event.reason: "{{parsed_event.message.reason}}"
           event.action: "{{parsed_event.message.action}}"
           host.ip: "{{parsed_event.message.host_ip}}"
-          host.hostname: "{{parsed_event.message.host_name}}"
-          host.name: "{{parsed_event.message.host_name}}"
+          host.hostname: "{{parsed_event.message.host_hostname}}"
+          host.name: "{{parsed_event.message.host_name or parsed_event.message.host_hostname}}"
           user.name: "{{parsed_event.message.username}}"
+          file.path: "{{parsed_event.message.file_path}}"
+          file.size: "{{parsed_event.message.file_size}}"
+          process.pid: "{{parsed_event.message.process_id}}"
+          process.executable: "{{parsed_event.message.process_path}}"
+          process.name: "{{parsed_event.message.process_path | basename}}"
+          process.working_directory: "{{parsed_event.message.process_path | dirname}}"
+      - translate:
+        dictionary:
+          "Blocked": ["denied"]
+        mapping:
+          event.action: event.type
+        fallback: ["info"]
+      - set:
+          process.args: "{{parsed_event.message.process_params.split(' ')}}"
```

```suggestion
          process.args: ["{{parsed_event.message.process_params.split(' ')}}"]
```

This is an array no ?